ui/appui-abstract/src/appui-abstract/properties/Description.ts

```diff
@@ -83,11 +83,10 @@ export interface PropertyDescription {
   editor?: PropertyEditorInfo;
   /** Information for a property converter */
   converter?: PropertyConverterInfo;
-  /** Quantity type key used to look up formatting and parsing specs. This is typically either the name of a quantity type used by a tool
-   *  or the full name of a KOQ (schema:koq).
-   * @alpha
+  /** KindOfQuantity full name used to look up [[FormatProps]]
+   * @beta
    */
-  quantityType?: string;
+  kindOfQuantityName?: string;
```

Comment:
This `@alpha` API was added many years ago, would it be safer to deprecate instead?

Reply:
Makes sense - I've kept quantityType back on, but marked it as deprecated.